bootstrap/components/sources/hdfs-source-topology-component.json

```diff
@@ -33,6 +33,24 @@
         "defaultValue": "/tmp/sourcefiles/",
         "hint": "schema"
       },
+      {
+        "uiName": "Schema Name",
+        "fieldName": "schemaName",
+        "isOptional": false,
+        "tooltip": "Name of the schema to which messages in the given `Source Dir` adhere to.",
+        "type": "string",
+        "defaultValue": "",
+        "hint": "schema"
+      },
+      {
+        "uiName": "Schema Version",
+        "fieldName": "schemaVersion",
+        "isOptional": false,
+        "tooltip": "Version of the given schema",
+        "type": "number",
+        "defaultValue": "-1",
+        "hint": "schema"
```

Comment:
Can we change schemaVersion field to match what we have in kafka source? Mainly type and hint

Reply:
Made the changes as suggested.